modal/verify-nft.js

```diff
@@ -63,6 +64,25 @@ module.exports = {
           const warning = warningEmbed('Verify Problem', "The BIP-322 node couldn't verify your signature.")
           return await interaction.editReply({ embeds: [warning], ephemeral: true })
         }
+
+        let userAddress = await UserAddresses.findOne({
```

Comment:
theres more than one location that we have our own `upsert` logic now - lets create our own DB `upsert` method that can be exported so that we can reduce code here and the other locations that we do it.